### ISSUE TITLE
Non-dismissable sidebars obscuring main content at 200%+ scaling / on mobile

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2409]
When a page involving a sidebar pattern is viewed on mobile or scaled to at least 200% on desktop, sidebars are either dismissed or fill the viewport, depending on user action. However, sidebars that are intended to be persistent and thus do not provide a close functionality (i.e. on Configure resource pages) fill viewports and cannot be dismissed, obscuring main content.
